parsely/parselyandroid/ParselyTracker.java

```diff
@@ -258,48 +392,34 @@ private String JsonEncode(Map<String, Object> map){
 
     /*! \brief Allow Parsely to send pageview events
     *
-    *  Instantiates the callback timer responsible for flushing the events queue.
+    *  Instantiates the callback flushTimer responsible for flushing the events queue.
     *  Can be called before of after `stop`, but has no effect if used before instantiating the
     *  singleton
     */
     public void setFlushTimer(){
-        if(this.flushTimerIsActive()){
-            this.stopFlushTimer();
-        }
-        this.timer = new Timer();
-        this.timer.scheduleAtFixedRate(new TimerTask(){
-            public void run(){
-                flush();
-            }
-        }, this.flushInterval * 1000, this.flushInterval * 1000);
+        this.flushManager.start();
```

Comment:
Yeah, I was torn on that because backwards compatibility. The manager is private now but setFlushTimer is public. I'll take a look again and maybe simply rename it to something like `startFlushTimer`.

Reply:
Since we're making this a `3.0.0` release and changing the public API anyway (see `trackURL`), backward compatibility isn't a concern IMO.